src/Controllers/Library/search.php

```diff
@@ -1,6 +1,54 @@
 <?php
-
+/**
+ * Allows URY Librarians to search for Tracks.
+ */
 use \MyRadio\MyRadio\CoreUtils;
+use \MyRadio\MyRadio\URLUtils;
+use \MyRadio\ServiceAPI\MyRadio_Track;
+use \MyRadio\MyRadio\MyRadioForm;
+use \MyRadio\MyRadio\MyRadioFormField;
+
+$form = new MyRadioForm(
+    'lib_search',
+    'Library',
+    'search',
+    [
+        'title' => 'Library Search',
+    ]
+);
+
+$form->addField(
+    new MyRadioFormField('title', MyRadioFormField::TYPE_TEXT, ['required' => false, 'label' => 'Title', 'placeholder' => 'Filter by track title...'])
+)->addField(
+    new MyRadioFormField('artist', MyRadioFormField::TYPE_TEXT, ['required' => false, 'label' => 'Artist', 'placeholder' => 'Filter by artist name...'])
+);
+        
+if ($_SERVER['REQUEST_METHOD'] === 'POST') {
+    //Submitted
+    $data = $form->readValues();
+
+    if (isset($data['title']) || isset($data['title'])) {
+        $tracks = MyRadio_Track::findByOptions(
+            [
+                    'title' => isset($data['title']) ? $data['title'] : '',
+                    'artist' => isset($data['artist']) ? $data['artist'] : '',
+                    'limit' => 30
```

Comment:
Well, I thought, since the tables paginate automatically by default at 10 per page, by the time you've read 3 pages and not found it, you would probably improve your search entry to narrow it down. I just did this because that's what stuff like Show Planner does when you search for a track.